Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+pipeline {
+  agent {
+    label 'sbt-slave'
+  }
+
+  environment {
+    // Ensure that build scripts recognise the environment they are running within
+    CI = ''
+    SCALA_VERSION = '2.10.6'
```

Comment:
I don't like allowing a different version of scala here than in the `build.sbt`. This isn't a library that needs to be crossbuilt, so this only causes duplication.